courses/github-for-devs-self.yml

```diff
@@ -16,7 +16,7 @@ sections:
       - DSS-03_issues
       - DSS-END-02_getting-started
   -
-    title: The Basic GitHub Workflow
+    title: The GitHub Flow pt. 1: Branching and Committing
```

Comment:
@1point618 replace all instances of GitHub Flow with GitHub Workflow. Also, remove the pt. 1 and pt. 2